test/server/compiler-test.js

```diff
@@ -971,4 +971,41 @@ describe('Compiler', function () {
             });
         });
     });
+
+    describe('Regression', function () {
+        it('Incorrect callsite line in error report on node v0.10.41 (GH-599)', function () {
+            this.timeout(5000);
+
+            var src      = 'test/server/data/regression/gh-599/testfile.js';
+            var compiler = new Compiler([src]);
+
+            return compiler
+                .getTests()
+                .then(function (tests) {
+                    var test = tests[0];
+
+                    return test.fn(testRunMock);
+                })
+                .then(function () {
+                    throw 'Promise rejection expected';
+                })
+                .catch(function (err) {
+                    var expectedCallsite =
+                            '   19 |};\n' +
+                            '   20 |\n' +
+                            "   21 |test('test', async () => {\n" +
+                            '   22 |    await obj\n' +
+                            '   23 |        .method2()\n' +
+                            ' > 24 |        .method1()\n' +
```

Comment:
No need to put whole callsite here. It's enough to check that it contains `  > 24 |        .method1()` string.